pages/common/git-rebase.md

```diff
@@ -1,23 +1,24 @@
 # git rebase
 
 > Apply local commits on top of another branch's history.
+> This effectively "moves" an entire branch to another base, by creating copies of the commits in the new location.
 
-- Rebase your local branch interactively with the latest changes in local master:
+- Rebase the current branch on top of the master branch:
 
-`git rebase -i master`
+`git rebase master`
 
-- Rebase your local branch interactively with the latest changes from upstream:
+- Start an interactive rebase, which allows the commits to be reordered, omitted, combined or modified:
 
-`git fetch origin; git rebase -i origin/master`
+`git rebase -i {{target_base_branch}}`
 
-- Handle an active rebase merge failure, after editing conflicting file(s):
+- Continue a rebase that was interrupted by a merge failure, after editing conflicting files:
 
 `git rebase --continue`
 
-- Abort a rebase in-progress:
+- Abort a rebase in progress (e.g. if it is interrupted by a merge conflict):
 
 `git rebase --abort`
 
-- Rebase your local branch by specifying new base commit and old base commit:
+- Rebase a branch starting from a specific base commit, rather than the common ancestor shared with the target branch:
 
 `git rebase --onto {{new_base_commit}} {{old_base_commit}}`
```

Comment:
@mhagger thanks for your input. I left this discussion alone for some time to see if I could come back to it with a fresh head. My thoughts after reviewing the whole discussion, drawing deveral diagrams and testing multiple command invocations and the corresponding descriptions, are:

- we should use the simplest possible `--onto` invocation, which is already advanced enough for an introductory cheatsheet like this. That means using as many defaults as possible -- e.g. rebasing the current branch so the third argument can be omitted.
- I'd rather not introduce the `..` notation, which is its own beast to describe succinctly.
- Instead of an accurate description, we should strive for one that could be easily conceptually grasped, while simultaneously mapping to a simple enough command invocation (this includes choosing between using a third branch as a starting base versus walking back from the current branch using `~`).

That said, my current proposal would be:
```
- Transplant the current branch from one base to another:

`git rebase --onto {{new_base}} {{old_base}}`
```

(We could use "move" instead of "transplant", but the "base" intuition may be slightly less evident in that case)

What do you guys think? @liuderchi, @mhagger